### PR TITLE
Add W3 Schools

### LINKED ISSUE #6134: W3School
**Brand Name: W3School**
**Website: https://www.w3schools.com/**
**Alexa rank:150**
  

**Official resources for icon and color:**
  

Please add [W3School](https://www.w3schools.com/) icon

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
w3schools

**Issue:** closes #6134 

**Similarweb rank:** 1,029. See https://www.similarweb.com/website/w3schools.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from header https://profile.w3schools.com/
Color: Taken from logo `#04aa6d`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13737,6 +13737,11 @@
             "hex": "D95E26",
             "source": "https://www.vyond.com"
         },
+        {
+            "title": "W3 Schools",
```
Comment (adamrusted) [inline _data/simple-icons.json:13741]:
```suggestion
            "title": "W3Schools",
```

It's stylized most places as `W3Schools` (one word) - could you update this, and the title in the SVG?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Fixed

**Comment by PeterShaggyNoble** [comment]
Before reviewing this for merging, I wonder could we get away with just the "w3" here, as seen in their favicon? What do you think, @PratyushJayachandran & @adamrusted?

**Comment by PeterShaggyNoble** [comment]
W3Schools isn't affiliated with the W3C.

**Comment by adamrusted** [comment]
> W3Schools isn't affiliated with the W3C.



We should be okay to include it then. @PratyushJayachandran - was there any wording regarding how they want their logo?

**Comment by PratyushJayachandran** [comment]
@PeterShaggyNoble @adamrusted We can but then I thought it might not represent the correct brand. Also to not confuse with W3C whose domain name is w3.org

**Comment by PeterShaggyNoble** [comment]
> Also to not confuse with W3C whose domain name is w3.org

Good point 👍🏻 I think, on that then, we should run with the full logo.

**Review by PeterShaggyNoble — APPROVED** [review]
Looks good to me, @PratyushJayachandran 👍🏻

There are a few duplicate points in the left of the 2 `s`s and the right of the `h` but my own efforts to remove them severely distorted the letters, so we'll have to run with them.

**Pull request merged**